clayui.com/src/pages/index.js

```diff
@@ -401,7 +403,7 @@ export default () => {
 
 							<div className="col-lg text-center text-lg-right">
 								<div>
-									{'Copyright © 2019 '}
+									{'Copyright © ' + `${CopyrightYear} `}
```

Comment:
ci might be complaining about the not using string templating for the "copyright" portion.
```suggestion
									{`Copyright © ${CopyrightYear}`}
```